packages/bitcore-client/bin/wallet-tx

```diff
@@ -9,22 +9,36 @@ program
   .option('--path <path>', 'REQUIRED - Where wallet is stored')
   .option(
     '--addresses <addresses>',
-    'REQUIRED - Address of receiver, or json array [{address: abc1234, satoshis: xxxxx}]'
+    'REQUIRED - Address of receiver, or json array \'[{"address": "abc1234", "satoshis": xxxxx}]\''
   )
   .option('--amount <amount>', 'REQUIRED - Value to transfer')
   .option('--fee <fee>', 'REQUIRED - The transaction fee to pay')
   .option('--utxos <utxos>', 'REQUIRED - Unspent transactions that can be spent')
   .option('--change <change>', 'REQUIRED - Change addresses')
   .parse(process.argv);
 
+if (!program.path || !program.amount || !program.addresses || !program.fee || !program.utxos || !program.change) {
```

Comment:
What if we made a thing that did something like 

```
for(option in program.options) {
  if(option.required && !program.option){
   console.error(`${option.name} is required to run this command`)
  }
}
```

Reply:
Then we could just make our own little wrapper around commander that did that

Reply:
#1618